typeDefs.js

```diff
@@ -76,6 +81,37 @@ const typeDefs = gql`
     tokenPass: String
   }
 
+  type OrderUser {
```

Comment:
Please, move all of the types and inputs related to the orders to the `order.graphql` file because currently `typeDefs` looks like a big rubbish bin, we need to clear it slowly

Reply:
fixed